python/ray/serve/schema.py

```diff
@@ -1042,3 +1042,22 @@ def _get_status(self) -> ServeStatus:
                 for app_name, app in self.applications.items()
             },
         )
+
+    def dict(self, *args, **kwargs) -> Dict[str, Any]:
```

Comment:
Public API: this docstring is too much for user to understand. (ref: https://docs.ray.io/en/master/serve/api/doc/ray.serve.schema.ServeInstanceDetails.html)

I would suggest to remove the all the internal implementation into the code comment.

Reply:
Oh, good call. let me see what else we can do here

Reply:
https://docs.ray.io/en/master/serve/api/doc/ray.serve.config.AutoscalingConfig.html#ray.serve.config.AutoscalingConfig
ah, is the serialized_policy_def supposed to be exposed to users? I didn't find test how we use it. Is user setting this field to control the policy?

Reply:
`policy` is the one user pass. `serialized_policy_def ` is used internally

Reply:
can we name this something more clear such as `get_user_facing_json_serializable_dict`? (if you have a more concise suggestion go for it :D)

Reply:
I believe if you change it to _serialized_policy_def, user will not read it from the documentation. (separate things :))

Reply:
@edoakes sg, will rename!

@sihanwang41 yep, I think that will be the case. There will be a few other attributes that need the underscore, will do those in the followup.